src/error.rs

```diff
@@ -81,6 +81,8 @@ pub enum ServiceError {
     Channel,
     #[error("no service")]
     NoService,
+    #[error("age {age}s > {max_age}s")]
```

Comment:
the naming of this new variant `Check` isn't obvious in context of the age-related error message.

Reply:
On the other hand, hyper-focused error variants can be cumbersome.

Reply:
Yeah.. I was being lazy and figured I'd leak this error from the `validator` feature to maybe use it later. There also wasn't a good way to feature flag the error entry out without the error attribute macro blowing up